### DISCUSSION TITLE
Was my website the victim of a targeted attack?

### DISCUSSION THREAD

**Discussion by sezze** [vercel/community#1996] [Help] [answered]
### Summary

My website was up for a few months without a new deployment, and suddenly I get emails saying that I've reached 75% of my usage, then 100%, then a ban, all within just a few minutes. There was an enormous spike in usage to one specific API endpoint, and nothing else. The site in question was literally a clone of the Vercel Next.js E-commerce project. So I am now temporarily banned and my actual websites like my portfolio are inaccessible.

I am on the free tier as these are non-commercial, but I still sent a support ticket, although it does mention that they typically don't respond to the free tier.

Here is my small personal website, which you can notice is down as a result: [aarnio.dev](https://aarnio.dev/)

<img width="608" alt="image" src="https://user-images.githubusercontent.com/9763776/230336877-bc7ef8e4-7669-4a5a-b055-42496a390af7.png">

**Comment by c-jardine** [comment]
I just had this happen too. The website in question isn’t even available because my client didn’t renew their domain. I’m glad to see I’m not the only one this happened to, but wtf is going on? I have a couple other client sites on my account and now they are inaccessible because my account is blocked.

I love Vercel, but knowing this might be a widespread problem is scary.

HELP US!

**Reply by sezze** [reply]
I got a response from Vercel, so I can at least get my account unblocked. This does not still explain what actually happened. Hopefully it's just some misconfiguration, although my code does not have a magical misconfiguration that would send 100k requests within a few minutes.

**Reply by c-jardine** [reply]
How did you get in touch with them? I sent them a message through Vercel last night when it happened and tagged them in a post on Twitter but still haven't heard back.

It's definitely gotta be an issue on their end. I haven't even touch Vercel in probably a week, and my charts look exactly like yours. Glad to hear you're getting your account unlocked!

**Reply by sezze** [reply]
I opened a ticket, Vercel dashboard > Help > Create a case

I chose Severity 2, as it's important but no one's dying or anything. And I thought "Abuse" was the closest problem area. Response in under 12 hours in my case.

**Reply by amyegan** [reply, marked as answer]
I don't have insight into your project logs from here, so I don't know what exactly caused this problem for you. But I want to share a pattern that I've seen here in the past in case it helps either of you. There are some services and configurations that can cause rapid re-try problems that cause a huge amount of requests. Things behave as expected for a while and then a service outage or expired token causes a cycle of timeouts and retries. 

If I remember correctly, certain next-auth configurations caused this problem for people in the past. I'm sure there are other things that could cause it as well. You may be able to find some more details about the error in the [logs](https://vercel.com/docs/concepts/observability/runtime-logs) that can help you figure out how to prevent it in the future.

@c-jardine You should [submit a case](https://vercel.com/help#issues) to the Customer Success Team for help with this if you have not already. That will allow you to privately communicate account information with the team to find a solution and get your account unblocked.

**Comment by peteralvex789** [comment]
There could be various reasons why your website was targeted by an attack. It could be due to the sensitive data that your website holds or the popularity of your website that makes it a lucrative target for hackers. Additionally, the attack could be a result of a personal grudge or vendetta against you or your website.

However, regardless of the reason, it's important to take steps to secure your website and prevent any future attacks. You can use tools such as firewalls and antivirus software to protect your website, and ensure that all software and plugins are up to date.

As an example, you can use [Lightleap Professional Editor]() to scan your website's code for any vulnerabilities and help you secure your website against potential attacks. Remember, the safety and security of your website should always be a top priority.

**Reply by sezze** [reply]
Another tip would be to not download something from some sketchy APK hosting site.

Fair to say that AI is here to stay, this sounds just like ChatGPT, so it probably scans through threads and writes stuff like this with actual context. Pretty cool and pretty sad at the same time.
